app/javascript/article-form/components/__tests__/Preview.test.jsx

```diff
@@ -97,7 +97,7 @@ describe('<Preview />', () => {
       />,
     );
 
-    expect(container.find('.crayons-article__tags').text()).toEqual(
+    expect(container.find('.spec-article__tags').text()).toEqual(
```

Comment:
cc: @Ridhwana . We'll need to convert this test if this gets merged before our Preact upgrade PRs.

Reply:
Will keep an 👀 out for it 👍🏽